README.md

```diff
@@ -48,7 +48,7 @@ OPTIONS
   -c, --config=config                              Path to percy config file
   -h, --allowed-hostname=allowed-hostname          Allowable hostname(s) to capture assets from
   -p, --port=port                                  [default: 5338] Port
-  -t, --network-idle-timeout=network-idle-timeout  [default: 50] Asset discovery network idle timeout (in milliseconds)
+  -t, --network-idle-timeout=network-idle-timeout  [default: 125] Asset discovery network idle timeout (in milliseconds)
```

Comment:
I think this whole file will automatically update before the bot releases a new version. And I believe it will also exclude those `See code:` sections.

Reply:
Yeah it does, it'll get updated again. I just ran `oclif-dev readme` locally too.

Forgot it was going to update as soon as its merged 🤦‍♂